src/parsers/installrdf.js

```diff
@@ -105,6 +109,40 @@ export default class InstallRdfParser {
       this._getTopLevelNodeByTag('em:bootstrap')) === 'true';
   }
 
+  _getApplication() {
+    var targetApplications = this._getTopLevelNodesByTag(
+      'em:targetApplication');
+    var applications = [];
+
+    for (let targetApplication of targetApplications) {
+      let application = {};
+      let descriptionNodes = Array.prototype.slice.call(
```

Comment:
Anywhere you're doing this you can use `this._makeArray` I see there's a couple of other places that could be used in this file too.